migrations/030-submission-form-name.js

```diff
@@ -0,0 +1,16 @@
+var db = require('../src/db').sequelize;
+
+module.exports = {
+  up: function() {
+    try {
+      return db.query(`
+			  ALTER TABLE submissions ADD formName VARCHAR(255) NULL AFTER userId;
+			`);
+    } catch(e) {
+      return true;
```

Comment:
Ik zou hier nog een `console.error` gooien zodat we het kunnen debuggen, indien nodig